benchmark/spmv/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 add_executable(spmv spmv.cpp)
 target_link_libraries(spmv ginkgo gflags rapidjson)
 if (GINKGO_BUILD_CUDA)
+    enable_language(CUDA)
+    set(CMAKE_CUDA_STANDARD 11)
+    set(CMAKE_CUDA_STANDARD_REQUIRED ON)
     target_compile_definitions(spmv PRIVATE HAS_CUDA=1)
-    target_link_libraries(spmv ginkgo ${CUDA_RUNTIME_LIBS}
-        ${CUBLAS} ${CUSPARSE})
-    target_include_directories(spmv SYSTEM PRIVATE ${CUDA_INCLUDE_DIRS})
```

Comment:
These were needed, as you can see from the failing shared library builds (that should be the root of the problem).